src/js/control-bar/picture-in-picture-toggle.js

```diff
@@ -27,13 +27,10 @@ class PictureInPictureToggle extends Button {
   constructor(player, options) {
     super(player, options);
     this.on(player, ['enterpictureinpicture', 'leavepictureinpicture'], this.handlePictureInPictureChange);
+    this.on(player, 'loadedmetadata', this.handlePiPEnabledChange);
```

Comment:
could listen here to `disablepictureinpicturechanged` (a custom event) as outlined in the player method.

Reply:
Thx, I added support for this event.